src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/RealExternalServiceDAO.scala

```diff
@@ -0,0 +1,43 @@
+package org.broadinstitute.dsde.rawls.dataaccess
+
+import akka.actor.ActorSystem
+import com.typesafe.config.Config
+import org.broadinstitute.dsde.rawls.model.AgoraEntity
+import org.broadinstitute.dsde.rawls.model.JobStatus
+import org.broadinstitute.dsde.rawls.model.WorkspaceJsonSupport._
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+import scala.util.Try
+import spray.client.pipelining._
+import spray.http._
+import spray.http.HttpHeaders.Cookie
+import spray.httpx.SprayJsonSupport._
+import spray.json._
+
+/**
+ * @author tsharpe
+ */
+class RealExternalServiceDAO( methodRepoConf: Config, execSvcConf: Config)( implicit system: ActorSystem ) extends ExternalServiceDAO {
+
+  override def getMethodConfig( namespace: String, name: String, version: String, authCookie: HttpCookie ): Try[HttpResponse] = {
```

Comment:
I don't think we should be passing in an HttpCookie but the credential itself. I don't like the api being http specific.

It may be idiomatic, but I am not sure I am a fan of returning Trys when any error will likely be unhandleable. It pollutes the code.

And why is this a Try[HttpResponse]. Seems weird to be returning the response as opposed to a MethodConfig.